libs/shared/ui/src/index.ts

```diff
@@ -11,3 +10,5 @@ export * from './lib/components/inputs/input-select-small/input-select-small'
 export * from './lib/components/navbar/navbar'
 export * from './lib/components/layouts/layout-page/layout-page'
 export * from './lib/components/button/button'
+export * from './lib/components/slider/slider'
```

Comment:
export should be like `./lib/components` and a `index.ts` file should be inlcuded in the folder exposing the components

Reply:
I don't agree with this approach, I thought we had already mentioned this
NX used the pattern /lib + /index.ts with all export when you use this field

<img width="369" alt="Capture d’écran 2022-04-05 à 15 30 30" src="https://user-images.githubusercontent.com/533928/161765011-3a5658fb-f8bf-4ed3-a522-10296ca6cedc.png">

I saw you added it for testing but I think we need to use the initial pattern
(cc: @WilliamTraoreee @bdebon)